assets/training/vision/components/image_classification/spec.yaml

```
@@ -188,7 +188,7 @@ outputs:
 
 code: ../src
 
-environment: azureml:AzureML-AutoML-DNN-Vision-GPU:115
+environment: azureml:automl-dnn-vision-gpu:@latest
```

Remove the colon when using @latest (applies to other files too):
```suggestion
environment: azureml:automl-dnn-vision-gpu@latest
```

---

Thanks for pointing out. Resolved.